cmd/apply.go

```diff
@@ -97,6 +98,20 @@ func RunApply(options *ApplyOptions) error {
 	}
 	logger.Info("Loaded cluster: %s", cluster.Name)
 
+	if options.Set != "" {
+		sets := strings.Split(options.Set, ",")
+		for _, set := range sets {
+			parts := strings.SplitN(set, "=", 2)
```

Comment:
We probably want to expand on this later and support more than just this syntax but for now it's okay!